deploy/mapping/base.mcf

```diff
@@ -208,13 +209,28 @@ typeOf: Source
 dcid: C:Source->id
 provenance: E:Source->E2
 domain: C:Source->domain
+name: C:Source->name
+url: C:Source->url
 functionalDeps: dcid
 
 Node: E:Source->E2
 typeOf: Provenance
 dcid: C:Source->prov_id
 functionalDeps: dcid
 
+Node: E:Dataset->E1
+typeOf: Dataset
+dcid: C:Dataset->id
+provenance: E:Dataset->E2
+name: C:Dataset->name
+url: C:Dataset->url
+isPartOf: C:Dataset->is_part_of
```

Comment:
Like above this should be E3 where E3 is a Source.

Reply:
done